.github/workflows/tests.yml

```diff
@@ -76,6 +88,21 @@ jobs:
           --health-timeout 5s
           --health-retries 5
 
+      openldap:
+        image: bitnami/openldap:2.5
+        ports:
+          - 1389:1389
+        env:
+          LDAP_ROOT: ${{ env.LDAP_ROOT }}
+          LDAP_PORT_NUMBER: ${{ env.LDAP_PORT_NUMBER }}
```

Comment:
this is not needed since 1389 is default port number and ports binding on line 94 are hardcoded anyway